src/main/java/com/amazon/dlic/auth/http/saml/HTTPSamlAuthenticator.java

```diff
@@ -76,7 +78,7 @@
     private HTTPJwtAuthenticator httpJwtAuthenticator;
     private Settings jwtSettings;
 
-    public HTTPSamlAuthenticator(final Settings settings, final Path configPath) {
+    public HTTPSamlAuthenticator(final Settings settings, final Path configPath) throws Exception {
```

Comment:
avoid throwing `Exception`, it should be either checked or run-time (preferred) Exception.

Reply:
I would prefer some exception/error to be thrown if the XML entered by the user is invalid. If I make this a checked/runtime exception I don't want it to silently fail.